src/celeritas/em/msc/UrbanMscHelper.hh

```diff
@@ -50,17 +50,19 @@ class UrbanMscHelper
     // TODO: the following methods are used only by MscStepLimit
 
     // The total energy loss over a given step length
-    inline CELER_FUNCTION Energy calc_eloss(real_type step) const;
-
-    // The kinetic energy at the end of a given step length corrected by dedx
-    inline CELER_FUNCTION Energy calc_end_energy(real_type step) const;
+    inline CELER_FUNCTION Energy calc_stopping_energy(real_type step) const;
```

Comment:
Again, this name is confusing as it sounds like something related to energy loss itself rather than the energy at the end of step taking into account for the energy loss by the step length.

Reply:
Sorry. Never mind.